aclocal.m4

```diff
@@ -20,6 +20,206 @@ You have another version of autoconf.  It may work, but is not guaranteed to.
 If you have problems, you may need to regenerate the build system entirely.
 To do so, use the procedure documented by the package, typically 'autoreconf'.])])
 
+# pkg.m4 - Macros to locate and utilise pkg-config.            -*- Autoconf -*-
+# serial 1 (pkg-config-0.24)
```

Comment:
Where did you get this license information from? Why is it added here?

Reply:
It was autogenerated with autobuild tools, I can delete it

Reply:
Please delete.

Reply:
removed